docs/concepts/abstractions/controllers/garbage-collection.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Garbage Collection
+---
+
+{% capture overview %}
+
+When you delete a Kubernetes object, the garbage collector deletes all
+of the objects' dependents. For example suppose you delete a ReplicaSet
+that owns several Pods. When you delete the ReplicaSet, the garbage collector
+deletes all Pods owned by the ReplicaSet.
+
+**Note**: Garbage collection is a beta feature and is enabled by default in
+Kubernetes version 1.4 and later.
+
+{% endcapture %}
+
+
+{% capture body %}
+
+## Owners and dependents
+
+Some Kubernetes objects are owners of other objects. For example, a ReplicaSet
+is the owner of a set of Pods. The owned objects are called *dependents* of the
+owner object. Every dependent object has a `metadata.ownerReferences` field that
+points to the owning API object.
+
+Sometimes, Kubernetes sets the value of `ownerReference` automatically. For
+example, when you create a ReplicaSet, Kubernetes automatically sets the
+`ownerReference` field of each Pod created for the ReplicaSet. You can also
+specify relationships between owners and dependents by manually setting the
+`ownerReference` field.
+
+## Controlling whether the garbage collector deletes dependents
```

Comment:
This title seems misleading.  The content implies that the garbage collector always deletes dependents, either synchronously or asynchronously.

Maybe

> ## Configuring garbage collection order
>
> By default, garbage collection will run against the dependent objects before the owning object is deleted.  When run this way, the deletion of both the owner and is dependents are asynchronous, resulting in a 200 OK response before completion.
> Alternatively, the owning object can be synchronously before running garbage collection by specifying `deleteOptions.orphanDependents=false`.  This will guarantee that the owning object is deleted before the 200 OK response is returned, however the dependent objects are deleted asynchronously after the owner.